tests/utils-e2e.ts

```diff
@@ -1031,3 +1037,147 @@ export const constructE2ETests = (
     });
   });
 };
+
+export const testGasEstimation = async (
+  network: Network,
+  srcToken: Token,
+  destToken: Token,
+  amount: bigint,
+  swapSide: SwapSide,
+  dexKeys: string | string[],
+  contractMethod: ContractMethod,
+  targetDifference?: number,
+) => {
+  assert(
+    testingEndpoint,
+    'Estimation can only be tested with testing endpoint',
+  );
+  // initialize pricing
+  const sdk = new APIParaswapSDK(network, dexKeys);
+  await sdk.initializePricing();
+  // fetch the route
+  const priceRoute = await sdk.getPrices(
+    srcToken,
+    destToken,
+    amount,
+    swapSide,
+    contractMethod,
+  );
+  console.log({ priceRoute });
+  // prepare state overrides
+  const tenderlySimulator = TenderlySimulatorNew.getInstance();
+  // any address works
+  const userAddress = ethers.Wallet.createRandom().address;
+  // init `StateOverride` object
```

Comment:
I suggest to use the same human readable so we used to this. should help us debugging

Reply:
added recognisable names in https://github.com/paraswap/paraswap-dex-lib/pull/886/commits/ed5740ba2baa5a3f78f5d516afa2e8203b22c493